packages/core/tdoptions.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
It can probably be done. But is it what we want to have docs under each package ? As it is now, it's all bundled under one directory easily tarable. Unless we want to keep a copy in git which resides with the package it documents.